modules/payment/src/PaymentMethodListBuilder.php

```diff
@@ -22,6 +23,16 @@ public function getFormId() {
     return 'commerce_payment_methods';
   }
 
+  protected function getEntityIds() {
+    /** @var RouteMatchInterface $route */
```

Comment:
Not sure how strict @bojanz is with coding standards, but this should be the FQN: 
`/** @var \Drupal\Core\Routing\RouteMatchInterface $route */`